src/stories/Card/Card.stories.tsx

```diff
@@ -1,23 +1,28 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react'
-
 import { Card } from './Card'
-import { CardBody } from './components/CardBody'
-import { CardFooter } from './components/CardFooter'
 
 export default {
     title: 'Components/Card',
     component: Card,
-    subcomponents: { CardBody, CardFooter },
-    argTypes: {
-        background: { control: 'color' },
-    },
 } as ComponentMeta<typeof Card>
 
-const Template: ComponentStory<typeof Card> = args => (
-    <Card {...args}>
-        <CardBody />
-        <CardFooter />
-    </Card>
-)
+const Template: ComponentStory<typeof Card> = args => <Card {...args} />
 
 export const Primary = Template.bind({})
+Primary.args = {
+    header: 'Card example',
+    content: (
+        <>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus
+            nec iaculis mauris. <a>@bulmaio</a>.<a href="#">#css</a>{' '}
+            <a href="#">#responsive</a>
+            <br />
+            <time dateTime="2016-1-1">11:09 PM - 1 Jan 2016</time>
+        </>
+    ),
+    footer: [
```

Comment:
This approach makes the footer less generic, but I assume we've arrived at some consensus over it